website/data/pages/accessibility.mdx

```diff
@@ -1,5 +1,36 @@
 ---
 title: Accessibility
+description: 
 ---
 
-# Title
+# Accessibility
+
+Making content available for everyone also passes by caring deeply about making content accessible for as many people as possible. A qualitative digital experience should have a  great User Experience (UX). This doesn't only include an appealing design and a clear copy but also the fact of being inclusive to everyone.
+
+## An inclusive usability
+
+It’s not completely true for now, but it’s an objective we’re aiming for. It’s important that Strapi product and plugins are usable and useful to everyone. <br/> <br/>
+To be inclusive means we must consider accessibility at the core of how we design and develop. <br/> <br/>
+Products and digital experiences must be accessible to everyone, including those with vision, hearing, cognitive, or mobility impairments.
+
+## Building experiences for everyone
+
+Using our components is a way to improve accessibility and consistency when building products for Strapi.
+
+## Assistive technology support
```

Comment:
That's not true. Even less with the rush of the v4. I would, personally, not put it here

Reply:
I agree !

Reply:
Removed it, thanks